_data/nav.yml

```diff
@@ -4,11 +4,11 @@
   url: /news/
 - title: Documentation
   url: /doc/
-- title: Google Group
-  url: https://groups.google.com/forum/#!forum/neovim
+- title: Community
+  url: /community/
 - title: Twitter
-  url: http://twitter.com/Neovim
-- title: Bountysource
+  url: https://twitter.com/neovim
+- title: Donations
```

Comment:
Actually, let's just remove this link. It has no need to be in the ever-present top navigation. The bitcoin/donate area below the fold on the mainpage is good enough. Anyone who wants to donate will have no trouble finding that.